libafl/src/mutators/encoded_mutations.rs

```diff
@@ -332,13 +332,12 @@ where
             }
         }
 
-        let other_size = state
-            .corpus()
-            .get(idx)?
-            .borrow_mut()
-            .load_input()?
-            .codes()
-            .len();
+        let other_size = {
+            let mut other_testcase = state.corpus().get(idx)?.borrow_mut();
```

Comment:
this pattern seems quite common, @domenukk maybe try to add a method to State to do that like we did for the metadata

Reply:
Cleaned this code up, thanks for the suggestion